src/corehost/common/pal.h

```diff
@@ -54,12 +54,15 @@
 #if defined(_WIN32)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
+#define BASE_HOST_RID _X("win")
```

Comment:
I understand. While this is straightforward for Windows (today), it is not so for even OSX. for example, we do not support 10.10 (even though our assets are in such folders). In 2.0, this will change to 10.12. In future, when Win7 retires, we may drop it as well. Linux is not different from the other two in this case.

Thus, we will need to keep this updated in the host. Base RID goal is for us to have something reasonable that when we dont know what platform we are on. Also, for us, falling to a consistent baseline makes things much easier to reason about.